build-support/docker/centos6/Dockerfile

```diff
@@ -4,18 +4,40 @@
 # Use centos6 for compatibility with glibc 2.12.
 FROM centos:6
 
-# Install python 2.7.13, (more) modern gcc, and a JDK.
+# Install a (more) modern gcc, and a JDK.
+# Also install sqlite-devel because python 3 errors on startup if not -- see
+# https://stackoverflow.com/a/39907500/2518889.
```

Comment:
We install more than just sqlite-devel for the sake of Pyenv. I'd reword the above line to be:

```
# Install a (more) modern gcc, a JDK, and dependencies for installing Python through Pyenv.